cmd/add.go

```diff
@@ -19,12 +19,18 @@ var addCmd = &cobra.Command{
 			wi := wallets[walletIndex]
 			wi.init()
 			var a *wallet.Account
-			for {
-				var err error
-				a, err = wi.w.NewAccount(nil)
+			var err error
+			if (walletAccountIndex >= 0) {
+				var index = uint32(walletAccountIndex)
```

Comment:
Use :=

Reply:
Using `var index := uint32(walletAccountIndex)` I get:

cmd/add.go:24:15: syntax error: unexpected :=, expecting =

Is that what you meant?

Reply:
No, when you use := you don't put `var` at the front.

Reply:
Done. Got it, thanks! (sorry, still learning golang).